scripts/smoke-access-browser.py

```
@@ -0,0 +1,337 @@
+        return
+
+
+def _bootstrap_cli(
+    data_root: Path, trailbase_root: Path, email: str, password: str
+) -> None:
+    server = http.server.ThreadingHTTPServer(("127.0.0.1", 8420), _Callback)
+    thread = threading.Thread(target=server.serve_forever, daemon=True)
+    thread.start()
+    pid, terminal = pty.fork()
+    if pid == 0:
+        os.execv(
```

Ruff (0.16.2)

[error] Starting a process without a shell

(S606)